var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -78,6 +84,8 @@ class Hypre(Package):
     depends_on("lapack")
     depends_on('superlu-dist', when='+superlu-dist+mpi')
 
+    conflicts('+cuda', when='+int64')
```

Comment:
Is this a conflict specific to _hypre_?  Or is it more generally applicable to `CUDA`?

Reply:
Currently, we can't use `--with-cuda` (`+cuda`) and `--enable-bigint` (`+int64`)  together in hypre. Hopefully, that will change in later versions.

Reply:
I don't know. I am going off of exawind repo package, which is known to work. @sayerhs - do you know?

Reply:
Is the issue with `hypre` or with the downstream apps `amr-wind` and `nalu-wind` ?

Does `hypre+cuda+int64` work in some cases, just not the way it is used in `amr-wind` and `nalu-wind` ?

Reply:
AMR-Wind/Nalu-Wind have no constraints... in fact we would like to be able to use `hypre+cuda+int64`